.github/workflows/python-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: Python integration tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  integration:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9, '3.10']
```

Comment:
Why only the last one is in quotes?

Reply:
Good point, I think we could quote them all... traditionally they weren't quoted and were treated as a float. So when 3.10 came out it was considered as 3.1 by the action.